indico/modules/events/registration/badges.py

```diff
@@ -209,3 +200,263 @@ def _build_pdf(self, canvas):
                 x_cm = (self.width - x*cm - self.tpl_data.width_cm*cm)
                 self._draw_badge(canvas, person, self.template.backside_template,
                                  self.backside_tpl_data, x_cm, y * cm)
+
+
+class GoogleWalletManager:
```

Comment:
This belongs in a separate file

Reply:
Ok, where do you want to put it / name it?
like indico/modules/events/google_wallet_manager.py ?

Reply:
how about `indico/modules/events/registration/google_wallet.py`? Anywy, easy to move it later, just want to not add something this huge to another file with unrelated stuff in it